docs/concepts/rfai.md

```diff
@@ -32,61 +32,65 @@ page_nav:
 
 ---
 
-The Request for AI portal is a platform that allows any member of the community to make requests for AI services to be published on the SingularityNET Registry.
-These requests are backed by AGI tokens, which are transferred to the developer who submits a valid solution for the request.
+The Request for AI Portal (RFAI) lets users make requests for AI services that they would like to see built and deployed onto the SingularityNET Network. Users can back requests with AGI tokens as a reward for having a service developed or as a pledge to make use of the service once it goes live. The AGI tokens are transferred to the developer who submits a valid solution for the request. 
```

Comment:
Are we going to support both pledges and rewards?

Reply:
Removed the word pledges. We just use the term backing for both requesting a service or backing an existing request